cuebot/src/main/java/com/imageworks/spcue/dispatcher/DispatchSupportService.java

```diff
@@ -389,6 +389,7 @@ public RunFrame prepareRqdRunFrame(VirtualProc proc, DispatchFrame frame) {
                         frame.command
                                 .replaceAll("#ZFRAME#", zFrameNumber)
                                 .replaceAll("#IFRAME#",  String.valueOf(frameNumber))
+                                .replaceAll("#OFRAME#",  String.valueOf(frameNumber+frame.chunkSize-1))
```

Comment:
This looks good to me. The only thing I'm unsure of is the name -- was `OFRAME` used elsewhere in the codebase or is this some kind of standard I'm unaware of?

I think it would be much more clear if we used `START_FRAME` and `END_FRAME` as our tokens. But of course we need to maintain `IFRAME` as it's what everyone is using already (and it makes sense for single-frame jobs). So I guess my suggestion is:

```
.replaceAll("#IFRAME#",  String.valueOf(frameNumber))
.replaceAll("#START_FRAME#",  String.valueOf(frameNumber))
.replaceAll("#END_FRAME#",  String.valueOf(frameNumber+frame.chunkSize-1))
```

Any thoughts from anyone else?

Reply:
Sounds much better, wasn't happy about the `OFRAME` anyway. :)

We could also add `CHUNK_FRAME` and `STEP_FRAME`. Or perhaps reverse them :
`FRAME_CHUNK`, `FRAME_STEP`, `FRAME_START`, `FRAME_END`

Reply:
I prefer reversing them to all be prefixed with `FRAME_`, and agree with adding `FRAME_CHUNK` and `FRAME_STEP`.